R/resource.R

```diff
@@ -265,8 +265,21 @@ hipercow_resources_validate <- function(resources,
                                         driver = NULL,
                                         root = NULL) {
 
+  # A bit of a hack for now - handle NULL driver (which often happens
+  # in the tests) - and also ignore if more than one driver is
+  # configured, which is caught elsewhere; here we would return a 
+  # "driver must be a scalar" error.
+  
+  if (is.null(driver) || length(driver) > 1) {
+    return(TRUE)
+  }
+  
   cluster_info <- hipercow_cluster_info(driver, root)
 
+  if (is.null(resources$queue$computed)) {
+    resources$queue$computed <- cluster_info$default_queue
```

Comment:
do we have this removed now from the configuration?

Reply:
Done now - along with valid_templates from clusters.R in the driver, which we no longer need